recipe/meta.yaml

```diff
@@ -12,7 +15,11 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 2
+  number: {{ number }}
+  string: "py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ number }}_{{ build_ext }}"
```

Comment:
@xhochy 
The number is also necessary for the build string. While I'm the first to be against useless templating, I think having `{{ number }}` makes a lot of sense in this recipe.